resources/dicts/relationship_events/normal_interactions/neutral.json

```diff
@@ -89,17 +84,14 @@
 		"id": "neutral_interaction4",
 		"interactions": [
 			"m_c stands tall when r_c walks by.",
-			"m_c doesn't mind r_c staring.",
 			"m_c tells r_c something that probably could've been kept private.",
-			"r_c sees m_c try and fail to something after claiming {PRONOUN/m_c/subject} {VERB/m_c/were/was} an expert at it.",
+			"r_c watches m_c try and fail at something after claiming {PRONOUN/m_c/subject} {VERB/m_c/were/was} an expert at it.",
 			"m_c puffs out {PRONOUN/m_c/poss} chest when r_c approaches.",
-			"m_c flashes a confident smile at r_c, but r_c doesn't know why.",
 			"r_c listens to m_c wax about how brave {PRONOUN/m_c/subject} {VERB/m_c/are/is}.",
 			"r_c is starting to doubt the truth behind m_c's epic story.",
 			"r_c doesn't bother to correct the blatantly wrong thing m_c said.",
 			"m_c does not take r_c's correction on something {PRONOUN/m_c/subject} said very well.",
 			"m_c falls flat on {PRONOUN/m_c/poss} face in front of r_c, but just gets right back up and keeps walking.",
-			"r_c compliments m_c about something, but m_c says {PRONOUN/m_c/subject} already knew that.",

```

Comment:
Removed

Reply:
Saving this one for a dislike increase interaction for "confident" trait ^^